### PR TITLE
update-readme

### PR DESCRIPTION
**Pull request by dezzyboy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,42 @@
 # BitClout Frontend
 This is the repository for the reference BitClout Angular app. It serves as an
 interface that allows users to interact with the data on the BitClout blockchain.
+## Getting started
+Bitclout frontend-ui depends on node Node.js, currently tested on v14.16.1.
+
+On MacOs  and Unbuntu, the most reliable way to install Node is to use the [Node Version Manager](https://github.com/nvm-sh/nvm) which allows you to have multiple node versions installed at once just incase you have other applications that depends on different versions.
```
Comment (HPaulson) [inline README.md:7]:
```suggestion
On MacOS and Linux based systems, the most reliable way to install Node is to use the [Node Version Manager](https://github.com/nvm-sh/nvm) which allows you to have multiple node versions installed at once just incase you have other applications that depends on different versions.
```

**Review by HPaulson — CHANGES_REQUESTED** [review]
If you're going to add these instructions, I'd also recommend adding such for Win-based systems.

File: README.md (hunk 1 above)
Comment (dezzyboy) [reply README.md]: thanks for the suggestions added windows node installation link..

**Review by dezzyboy — COMMENTED** [review]
made relevant changes

**Review by HPaulson — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,8 +1,42 @@
 # BitClout Frontend
 This is the repository for the reference BitClout Angular app. It serves as an
 interface that allows users to interact with the data on the BitClout blockchain.
+## Getting started
+Bitclout frontend-ui depends on node Node.js, currently tested on v14.16.1.
```
[the remark this answers is not in the document]
Comment (HPaulson) [reply README.md:6]: @dezzyboy Let me know what you think about this suggestion!

**Review by HPaulson — APPROVED** [review]

**Review by dezzyboy — COMMENTED** [review]
looks like this can be marked complete

**Comment by maebeam** [comment]
i simplified some of your changes. thanks!

**Pull request merged**